README.md

```diff
@@ -174,25 +174,27 @@ You may now set quotas per worker:
 ```ruby
 client = Redis.new(semian: {
   name: "inventory",
-  quota: 0.5,
+  quota: 0.49,
+  min_tickets: 2,
   success_threshold: 2,
   error_threshold: 4,
   error_timeout: 20
 })
 
 ```
 
-Per the above example, you no longer need to care about the number of tickets.
+Per the above example, you no longer need to care about the number of tickets. Rather, the tickets shall be computed as a proportion of the number of active workers.
 
-Rather, the tickets shall be computed as a proportion of the number of active workers.
+In this case, we'd allow 49% of the workers on a particular host to connect to this redis resource.
 
-In this case, we'd allow 50% of the workers on a particular host to connect to this redis resource.
+In particular, 1 worker = 1 ticket (due to `ceil`), 2 workers = 2 tickets (due to `min_tickets`), 4 workers = 2 tickets, 16 workers = 8 tickets, 100 workers = 49 tickets.
```

Comment:
`1 worker = 2 tickets`, due to `min_tickets`, right? Perhaps call out `ceil` for 4/16 workers.

Reply:
No, `1 worker = 1 tickets` due to `min(workers, max(tickets, min_tickets))`.
So `min(1, max(1, 2))` is 1.

Reply:
But I called out `ceil` for the other case.